libraries/ms-common/test/test_interrupt.c

```diff
@@ -0,0 +1,25 @@
+#include "interrupt.h"
+
+#include <stdbool.h>
+
+#include "unity.h"
+
+void setup_test(void) { }
+
+void teardown_test(void) {
+  // After a test concludes forcibly enable interrupts if they aren't already enabled.
+  interrupt_enable(false);
```

Comment:
I think it makes more sense to return if interrupts were disabled (i.e. were previously enabled and are now enabled).
Then this would be `interrupt_enable(true)` which is more obvious imo.